### ISSUE TITLE
Dropdowns Are not Scrollable

### ISSUE THREAD

**Issue by SpartanLines** [react-materialize/react-materialize#625]
### Description

The dropdown component is not scrollable. If there are too many items within, the moment you press to scroll the dropdown dissappears. The issue only happens on mobile view/devices. Works fine on desktop since the scrollbar is thick and always visible. Dropdowns are scrollable in Materialize , but something goes wrong with react-materialize. I've tried to look into the lib and spent the entire day searching for a fix or a workaround with no use.

### Steps to Reproduce

Here's a sample create-react-app with one component ( App.js) that will reproduce the error:

`import React, { Component } from 'react';
import {Dropdown,Navbar,NavItem} from 'react-materialize'
import './App.css';

`class App extends Component {`
  `render() {` 
   ` return (`
      `<div className="App">`
        `<ul id="dropdown1" className="dropdown-content"></ul>`   
         `   <Navbar center>`
         `   <Dropdown trigger={<NavItem>Drop me <i className="material-icons right">arrow_drop_down</i></NavItem> }>`
                  `  <li><NavItem href="#section-1" >section 1</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 2</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 3</NavItem></li>`
                    `<li><NavItem href="#section-1" >section 4</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 5</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 6</NavItem></li>`
                    `<li><NavItem href="#section-1" >section 7</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 8</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 9</NavItem></li>`
                    `<li><NavItem href="#section-1" >section 10</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 11</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 12</NavItem></li>`
                    `<li><NavItem href="#section-1" >section 13</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 14</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 15</NavItem></li>`
                    `<li><NavItem href="#section-1" >section 16</NavItem></li>`
                    `<li><NavItem href="#section-2" >section 17</NavItem></li>`
                    `<li><NavItem href="#section-3" >section 18</NavItem></li>`
                   `</Dropdown>`
            `</Navbar>`
      `</div>  `
    `);`
  `}`
`}`
`export default App;`

**Expected behavior:** To be able to scroll down the list

**Actual behavior:** the list disapppears the moment i try to scroll
### Versions

**react-materialize:** 2.3.3
**materialize-css:** 0.98.0
**react:** 16.4.1

**Comment by alextrastero** [comment]
Hey @SpartanLines, have you tried without `react-materialize`? Just using the materialize library does this not happen? I'm not sure this our issue

**Comment by SpartanLines** [comment]
Hey @alextrastero 
Yes, I've tried long dropdowns in materialize and they're definitely scrollable. I made a simple nav bar on an html page and everything worked correctly. I've also tried using a Material-UI menu and it works fine on react, so I don't think it's react's problem either.

I compared both the compiled HTML from react-materialize and the working materialize code. My guess (which could be wrong) is that something goes wrong with the data-activates attribute, whereas data-target works fine on materialize. Either that, or a certain css style is missing (i've tried playing with styles and nothing worked). I also tried to convert the data-activates into a data-target attribute, but I failed to place an active class on the list to display it. Then again, this is just my guess, hopefully, somebody can fix this or shed some light over what needs to be done or where the issue is.
